web/profiles/custom/os2loop/themes/os2loop_theme/templates/content/field--node--os2loop-shared-tags--os2loop-post.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
```

Comment:
This template should be renamed to `field--node--os2loop-shared-tags.html.twig` (i.e. without the content type name) as it will be reused by most content types. Then we can make more specific templates afterwards if need be.